crafters/image/ImageTorchTransformation/requirements.txt

```diff
@@ -1 +1,2 @@
 torchvision==0.9.1
+opencv_python==4.5.1
```

Comment:
I am not sure if it's ok to have this in the requirements as it is only used in the test and not anywhere in the crafter. There was a build error stating cv2 was not found. However, I do not see other crafters having OpenCV in requirements. Let me know.